mirrord/config/src/target.rs

```diff
@@ -290,6 +293,19 @@ pub struct PodTarget {
     pub container: Option<String>,
 }
 
+impl Display for PodTarget {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_fmt(format_args!(
+            "{}{}",
+            self.container
+                .clone()
```

Comment:
```suggestion
                .as_ref()
```
No need to clone the string, I think